src/content/private-keys-passwords/how-do-mnemonic-phrases-work.md

```diff
@@ -0,0 +1,53 @@
+{
+"title"       : "How do (encrypted) mnemonic phrases work?",
+"sort"        : "40",
+"category"    : "Private Keys & Passwords",
+"description" : "Private Keys & Passwords",
+"date_published" : "2018-06-20T08:00:00+08:00",
+"date_modified"  : "2018-06-20T08:00:00+08:00"
+}
+
+---%
+
+A **mnemonic phrase** or **mnemonic seed** is a set of typically either 12 or 24 words, which can be used to derive an infinite number of wallets. Often times these phrases are used by cryptocurrency hardware wallets, to be written down on a piece of paper by the user to safely back up the users' funds.
+
```

Comment:
can you link to how users should backup their wallets :)

Reply:
Thanks Sharon, added it :+1: